apps/graphql/src/definitions/Owner.ts

```diff
@@ -149,6 +162,16 @@ export const resolvers = {
       }
     },
     name: (owner: Owner) => owner.name || owner.login,
+    clientReferenceId: async (owner: Owner) => {
+      const account = await getOwnerAccount(owner);
+      if (account && account.stripeCustomerId) {
+        return account.stripeCustomerId;
+      }
+      if (account && account.id) {
+        return `account-${account.id}`;
+      }
+      return `${owner.type()}-${owner.id}`;
+    },
```

Comment:
Factorise `parse` and `format` of clientReferenceId in the same file, the logic is separated in two files it is complicated to maintain it actually.

Reply:
It will be a JSON with this interface:

```ts
type ClientReferenceID  = { accountId: string, stripeCustomerId: string | null, purchaserId: string }
```

We will encode it in base64 to obtain an ID.